src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteSkinPresence.java

```diff
@@ -0,0 +1,43 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFliteObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+public class TestAdsFliteSkinPresence extends TemplateNoFirstLoad {
+
+  private final String[] fliteSyntheticSuperheroesIframes = new String[]{
+      "iframe[id$=TOP_LEADERBOARD_0]",
+      "#f_ad_dd9465c8-a687-46cd-8550-b6ee529f76ed",
+      ".flite-ad"
+  };
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "fliteSkin",
+      groups = "AdsFliteSkinPresenceOasis"
+  )
+  public void AdsFliteSkinPresence(String wikiName,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)